app/controllers/api/v1/merchants/random_controller.rb

```diff
@@ -0,0 +1,5 @@
+class Api::V1::Merchants::RandomController < ApplicationController
+  def show
+    render json: Merchant.order("RANDOM()").first
+  end
```

Comment:
@riverswb definitely a great find! let's move this method out to the model though. maybe a method in our `Merchant` model that's called `random`?